subworkflows/local/bam_sentieon_dedup/main.nf

```diff
@@ -0,0 +1,48 @@
+//
+// SENTIEON DEDUP
+//
```

Comment:
```suggestion
```
The standard comment about "when clause condition is defined in the `conf/modules.config`" doesn't apply here. Let's just delete it.